00-init.tf

```diff
@@ -3,7 +3,7 @@ terraform {
   required_providers {
     azurerm = {
       source                = "hashicorp/azurerm"
-      version               = "3.10.0"
+      version               = ">= 3.15.0"
```

Comment:
All consumers need upgrading first,

e.g. CFT is on an older version:
https://github.com/hmcts/aks-cft-deploy/blob/111432387bb03fa8d647211fc60389ea72fce4bd/components/aks/init.tf#L10

Reply:
I'm reverting anyway, didn't need to upgrade